crates/web-plugins/did-endpoint/src/util.rs

```diff
@@ -0,0 +1,31 @@
+use did_utils::didcore::Document;
+use std::borrow::Cow;
+
+pub(crate) fn handle_vm_id<'a>(vm_id: &'a str, diddoc: &Document) -> Cow<'a, str> {
+    if vm_id.starts_with('#') {
+        Cow::Owned(diddoc.id.to_owned() + vm_id)
+    } else {
+        Cow::Borrowed(vm_id)
+    }
+}
```

Comment:
This function is a hack to bypass certain constraints of the `did:peer` method specification. Its purpose is to uniquely identify the keys used to generate a Peer DID address.

Reply:
@Hermann-Core I think what @Christiantyemele was saying here is that you should put the comment above or beside the code. So that someone else who might be working or come across these would understand what you doing. Is that right @Christiantyemele ?